service/test/lib/unit/configAuthTest.js

```diff
@@ -0,0 +1,100 @@
+    expect(configAuth.Action.ReadOrganization).to.equal('authorization:organizations:read')
+    expect(configAuth.Action.DeleteOrganization).to.equal('authorization:organizations:delete')
+    expect(configAuth.Action.ListOrganizations).to.equal('authorization:organizations:list')
+    expect(configAuth.Action.AllOrganization).to.equal('authorization:organizations:*')
+    expect(configAuth.Action.CreateTeam).to.equal('authorization:teams:create')
+    expect(configAuth.Action.UpdateTeam).to.equal('authorization:teams:update')
+    expect(configAuth.Action.ReadTeam).to.equal('authorization:teams:read')
+    expect(configAuth.Action.DeleteTeam).to.equal('authorization:teams:delete')
+    expect(configAuth.Action.ListTeams).to.equal('authorization:teams:list')
+    expect(configAuth.Action.ManageTeams).to.equal('authorization:teams:manage')
+    expect(configAuth.Action.AddTeamPolicy).to.equal('authorization:teams:policy:add')
+    expect(configAuth.Action.ReplaceTeamPolicy).to.equal('authorization:teams:policy:replace')
+    expect(configAuth.Action.RemoveTeamPolicy).to.equal('authorization:teams:policy:remove')
+    expect(configAuth.Action.AddTeamMember).to.equal('authorization:teams:user:add')
+    expect(configAuth.Action.ReplaceTeamMember).to.equal('authorization:teams:user:replace')
+    expect(configAuth.Action.RemoveTeamMember).to.equal('authorization:teams:user:remove')
+    expect(configAuth.Action.AllTeam).to.equal('authorization:teams:*')
+    expect(configAuth.Action.CreateUser).to.equal('authorization:users:create')
+    expect(configAuth.Action.UpdateUser).to.equal('authorization:users:update')
+    expect(configAuth.Action.ReadUser).to.equal('authorization:users:read')
+    expect(configAuth.Action.DeleteUser).to.equal('authorization:users:delete')
+    expect(configAuth.Action.ListUsers).to.equal('authorization:users:list')
+    expect(configAuth.Action.AddUserPolicy).to.equal('authorization:users:policy:add')
+    expect(configAuth.Action.ReplaceUserPolicy).to.equal('authorization:users:policy:replace')
+    expect(configAuth.Action.RemoveUserPolicy).to.equal('authorization:users:policy:remove')
+    expect(configAuth.Action.AllUser).to.equal('authorization:users:*')
+    expect(configAuth.Action.CreatePolicy).to.equal('authorization:policies:create')
+    expect(configAuth.Action.UpdatePolicy).to.equal('authorization:policies:update')
+    expect(configAuth.Action.ReadPolicy).to.equal('authorization:policies:read')
+    expect(configAuth.Action.DeletePolicy).to.equal('authorization:policies:delete')
+    expect(configAuth.Action.ListPolicies).to.equal('authorization:policies:list')
+    expect(configAuth.Action.AllPolicy).to.equal('authorization:policies:*')
+    expect(configAuth.Action.CheckAccess).to.equal('authorization:authn:access')
+    expect(configAuth.Action.ListActions).to.equal('authorization:authn:actions')
+
+    done()
```

Comment:
I find this unnecessary: we are basically checking that we correctly created the config file?